runtime/plugin/man.vim

```diff
@@ -1,6 +1,11 @@
-if get(g:, 'loaded_man', 0)
+if exists('g:loaded_man')
   finish
 endif
 let g:loaded_man = 1
 
-command! -count=0 -nargs=+ Man call man#get_page(<count>, <f-args>)
+let g:man_find_window = get(g:, 'man_find_window', 1 )
+
+command! -count=10 -complete=customlist,man#complete -nargs=* Man call
```

Comment:
In `man.vim` count is set to 0 and used as a extra argument. Two problems with this approach. First, because count is only a number you can never use a letter in the section names. `Man 1ssl cms` will throw an error on OSX even though it is a valid manpage because 3 extra arguments were sent to man.vim: `Man 0 1ssl cms`. If you use a number section it looks like `Man 3 printf`. 

Second, this default value of 0 on count is used to indicate an empty section. On some systems 0 is actually a manpage (https://en.wikipedia.org/wiki/Man_page#Manual_sections) so you can never actually specify manpage 0 because it will always be interpreted as empty.

To solve the first problem I made count a separate parameter. To solve the second problem I've made the default value of count 10 but imo that's an ugly solution. Anyone have a better idea?